sympy/sets/tests/test_sets.py

```diff
@@ -831,3 +831,12 @@ def test_Eq():
 
     assert Eq(s1*s2, s1*s2)
     assert Eq(s1*s2, s2*s1) == False
+
+def test_SymmetricDifference():
+   assert SymmetricDifference(FiniteSet(0,1,2,3,4,5), FiniteSet(2,4,6,8,10)) == FiniteSet(0, 1, 3, 5, 6, 8, 10)
+   assert SymmetricDifference(FiniteSet(2,3,4), FiniteSet(2,3,4,5)) == FiniteSet(5)
+   assert FiniteSet(1,2,3,4,5) ^ FiniteSet(1,2,5,6) == FiniteSet(3,4,6)
+
+   #Not Implemented yet
```

Comment:
You can create a test function with the `@xfail` decorator. `xfail` implies `expected to fail`.